opm/autodiff/StandardWell.hpp

```diff
@@ -220,6 +228,9 @@ namespace Opm
         // the saturations in the well bore under surface conditions at the beginning of the time step
         std::vector<double> F0_;
 
+        const RateConverterType& rateConverter_;
+        int pvtRegionIdx_;
```

Comment:
Is there any reason for not moving the scalingFactor method to the base class as well?

Reply:
I think the only possible reason was that MSW does not support solvent yet.  But it should have been moved while you can consider it as an oversight (there are probably more).  The combination of solvent and MSW is guarded in the constructor already. Now I think it can be moved to base class. Thanks for pointing it out. It is your call whether to address it.

Reply:
I can try. If it is more involving than anticipated I will suggest that we do this in a separate PR.

Reply:
Sure. Anyway, it is not the purpose of this PR and it is good that you pointed it out and some potentially duplicated code can be removed.

Reply:
Done.